.travis.yml

```diff
@@ -72,7 +72,6 @@ jobs:
   allow_failures:
     - os: osx
     - go: tip
-    - env: CROSS_PLATFORM=true
```

Comment:
You should probably delete the `CROSS_PLATFORM` variables above as well; I don't think they do anything other than drive this switch.

Reply:
done